server/cli/databases/postgres/change-database.cli

```diff
@@ -0,0 +1,9 @@
+/subsystem=datasources/data-source=KeycloakDS: remove()
+/subsystem=datasources/data-source=KeycloakDS: add(jndi-name=java:jboss/datasources/KeycloakDS,enabled=true,use-java-context=true,use-ccm=true, connection-url=jdbc:postgresql://${env.POSTGRES_PORT_5432_TCP_ADDR}:${env.POSTGRES_PORT_5432_TCP_PORT:5432}/${env.POSTGRES_DATABASE:keycloak}, driver-name=postgresql)
+/subsystem=datasources/data-source=KeycloakDS: write-attribute(name=user-name, value=${env.POSTGRES_USERNAME:keycloak})
```

Comment:
Hi @stianst,
You changed here env. variable for DB user. This variable was named POSTGRE_USER before (see [changeDatabase.xsl](https://github.com/jboss-dockerfiles/keycloak/blob/9dab19310bca1f50cf59492b8030e45f578830f4/server/databases/postgres/changeDatabase.xsl#L14). This change broke our KeyCloak configuration.
Can you change this variable name back, or is there any reason why was this variable change?

Regards, Jakub.